logs/plugin_bot.go

```diff
@@ -82,6 +82,13 @@ var cmdWhois = &commands.YAGCommand{
 		}
 
 		member := commands.ContextMS(parsed.Context())
+		memberCPY := parsed.GS.MemberCopy(true, member.ID)
+		if memberCPY == nil {
```

Comment:
Maybe replace it with `if memberCPY != nil .... member = memberCPY` and drop the else clause entirely?